src/pages/project/helpers/projectRoleHelpers.ts

```diff
@@ -8,6 +14,10 @@ export const isNonProjectManagerRole = (role: ProjectContributorRole) => {
   return role.type !== 'ProjectManager';
 };
 
+export const hasEmptyAffiliation = (role: ProjectContributorRole) => {
+  return role.affiliation?.id === '' || role.affiliation?.id === undefined;
```

Comment:
Blir det samme, men ser kanskje enklere og mer lesbart ut? Opp til deg :)

```suggestion
  return !role.affiliation?.id;
```